db/seeds.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# This file should contain all the record creation needed to seed the database with its default values.
+# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+#
+# Examples:
+#
+#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
+#   Character.create(name: "Luke", movie: movies.first)
+Message.create(message: 'Hello, world!')
+Message.create(message: 'How are you doing?!')
+Message.create(message: 'Bonjour!')
+Message.create(message: 'Welcome!')
+Message.create(message: 'Willkommen!')
```

Comment:
- [ ] You do not have any table with the name `Message`, instead you have `Greetings`. You have used this `Message` in your `seed.rb` file, controller, route file, and model. Rails system is based on naming convention over configuration. I suggest you use one particular name across your Model and Controller. Please fix this

As a result, I could not test if both front-end and backend apps are working well together, the next reviewer will confirm this once you fix all these issues I have raised and any other issues you might find.